src/sentry/integrations/bitbucket/client.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import absolute_import
+
+import datetime
+import jwt
+
+from unidiff import PatchSet
+
+from six.moves.urllib.parse import urlparse
+
+from sentry.utils.http import absolute_uri
+from sentry.integrations.atlassian_connect import get_query_hash
+from sentry.integrations.client import ApiClient
+
+BITBUCKET_KEY = '%s.bitbucket' % urlparse(absolute_uri()).hostname
+
+
+class BitbucketAPIPath(object):
+    """
+    All UUID's must be surrounded by culybraces.
+    repo_slug - repository slug or UUID
+    username - username or UUID
+    """
+
+    issue = u'/2.0/repositories/{username}/{repo_slug}/issues/{issue_id}'
```

Comment:
why did you change username and repo slug to be separate? it looks like in the old client they are together https://github.com/getsentry/sentry-plugins/blob/master/src/sentry_plugins/bitbucket/client.py#L34 and a lot of our existing repo / commit functionality depends on their being together

Reply:
@macqueen that's not the same api. I'm using 2.0 rather than 1.0. If repo_slug actually refers to the username and the slug, it probably should be changed/renamed as that is unclear. I assumed the repo slug only contained the repo slug.

Reply:
ok, feel free to rename it, but the rest of our code combines these so the client is going to have to work with the combined name

Reply:
<img width="601" alt="screenshot 2018-05-23 14 39 31" src="https://user-images.githubusercontent.com/5026776/40452569-1ff4a8a4-5e97-11e8-9286-3adb2df0183c.png">

Reply:
So, just so I'm clear the repo_name in this case is username/repo_slug correct? in your example getsentry is the username and sentry is the repo_slug?

Reply:
yeah, i think it could also probably be `team_slug/repo_slug`? But if you look at the v1 docs here: https://confluence.atlassian.com/bitbucket/issues-resource-296095191.html#issuesResource-GETanindividualissue
we turn this:
`https://api.bitbucket.org/1.0/repositories/{accountname}/{repo_slug}/issues/{issue_id}`
into:
`'/1.0/repositories/%s/issues/%s' % (repo, issue_id)`

so yeah, sorry if that wasn't clear, but by `repo` we mean the whole identifier